util/common.go

```diff
@@ -67,3 +79,16 @@ func WrapErr(err error, kinds ...*errors.Kind) error {
 	}
 	return err
 }
+
+// SplitStringMap splits map[string]string to arrays of keys and arrays of values
+func SplitStringMap(m map[string]string) ([]string, []string) {
+	var (
+		keys   []string
+		values []string
+	)
+	for k, v := range m {
+		keys = append(keys, k)
+		values = append(values, v)
+	}
+	return keys, values
```

Comment:
Just I want to be sure that it's OK that order of returned slices can be random. in other words sometimes `SplitStringMap(mymap) != SplitStringMap(mymap)`

Reply:
I think it's ok because:
- there're tags and order is not crucial
- keys and values order are preserved